src/scripts/main.js

```diff
@@ -18,10 +18,27 @@ const food = {
   },
 };
 
+const treeDiv = document.createElement('div');
+
+treeDiv.id = 'tree';
+document.body.appendChild(treeDiv);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
```

Comment:
You are using `Object.keys(data[key]).length > 0` to check if the object is empty. While this works, a more efficient way would be to use `Object.keys(data[key]).length`. This way, you don't have to calculate the length of the keys array just to check if it's empty.